package.json

```diff
@@ -94,11 +94,12 @@
     "zone.js": "~0.11.4"
   },
   "resolutions": {
-    "**/wait-on/axios": "^0.27.2",
     "**/chokidar/glob-parent": "^6.0.2",
     "**/@angular-devkit/build-angular/minimatch": "^3.1.2",
     "**/@angular-devkit/build-angular/babel-loader": "^8.3.0",
     "**/@angular-devkit/build-angular/loader-utils": "^2.0.4",
+    "**/@angular-devkit/build-angular/webpack-dev-server": "^4.15.2",
```

Comment:
`yarn list` seems to list only that fsevents@1.2.13 is used by webpack-dev-server@3.11.3. why are the other updates needed?

Reply:
```jared:angular$ yarn why fsevents
yarn why v1.22.19
[1/4] 🤔  Why do we have the module "fsevents"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "fsevents@2.3.2"
info Reasons this module exists
   - "_project_#@angular#compiler-cli#chokidar" depends on it
   - Hoisted from "_project_#@angular#compiler-cli#chokidar#fsevents"
   - Hoisted from "_project_#ng-packagr#rollup#fsevents"
   - Hoisted from "_project_#jest#@jest#core#jest-haste-map#fsevents"
info Disk size without dependencies: "164KB"
info Disk size with unique dependencies: "164KB"
info Disk size with transitive dependencies: "164KB"
info Number of shared dependencies: 0
=> Found "@okta/test.e2e#fsevents@2.3.2"
info Reasons this module exists
   - "_project_#@okta#test.e2e#@wdio#cli#chokidar" depends on it
   - Hoisted from "_project_#@okta#test.e2e#@wdio#cli#chokidar#fsevents"
   - in the nohoist list ["/_project_/test/apps/*","/_project_/**/@wdio/**"]
info Disk size without dependencies: "164KB"
info Disk size with unique dependencies: "164KB"
info Disk size with transitive dependencies: "164KB"
info Number of shared dependencies: 0
✨  Done in 0.32s.
```

Key line: `- Hoisted from "_project_#@okta#test.e2e#@wdio#cli#chokidar#fsevents"`

When possible we upgrade the dependency which requires the vulnerable package. Essentially when jump major versions, it's possible to break functionality when forcing a newer version of a package upon a library that is expecting a older one.

`@angular-devkit` ultimately depends on `webpack-dev-server` which depended on `fsevents@1.2.13`